pkg/api/index.go

```diff
@@ -623,10 +625,16 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
 	themeURLParam := c.Query("theme")
 	if themeURLParam == lightName {
 		data.User.LightTheme = true
+		data.User.FusebitTheme = false
```

Comment:
Let's add data.user.DarkTheme = false and, lower, = True so that each of the three themes is represented.